specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/security.json

```diff
@@ -2491,6 +2922,251 @@
+                }
+            }
+        },
+        "NorthSouthHardeningsAlert": {
+            "description": "A north-south hardening alert",
+            "properties": {
+                "properties": {
+                    "description": "Properties of the north-south hardening alert",
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/NorthSouthHardeningsAlertProperties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/Resource"
+                }
+            ]
+        },
+        "NorthSouthHardeningsAlertList": {
+            "description": "Response for ListNorthSouthHardeningsAlert API service call. Retrieves all north-south hardening alerts that belongs to a north-south hardening resource",
+            "properties": {
+                "value": {
+                    "description": "The north-south hardening alerts of a north-south hardening resource",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/NorthSouthHardeningsAlert"
+                    }
+                },
+                "nextLink": {
+                    "description": "The URL to get the next set of results",
+                    "type": "string"
+                }
+            }
+        },
+        "NorthSouthHardeningsPolicyRule": {
+            "description": "A north-south hardening policy rule",
+            "properties": {
+                "name": {
+                    "description": "A policy rule name",
+                    "type": "string"
+                },
+                "access": {
+                    "description": "A policy rule access",
+                    "type": "string",
+                    "enum": [
+                        "Allow",
+                        "Deny"
+                    ],
+                    "x-ms-enum": {
+                        "name": "accees",
+                        "modelAsString": true
+                    }
+                },
+                "destinationPort": {
+                    "description": "A policy rule detination port",
+                    "$ref": "#/definitions/PortNumber"
+                },
+                "protocols": {
+                    "description": "A policy rule protocol",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/NorthSouthProtocol"
+                    }
+                },
+                "sourceAddresses": {
+                    "description": "A policy rule source address",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                }
+            }
+        },
+        "NorthSouthHardeningsPolicyProperties": {
+            "description": "A north-south hardening policy",
+            "properties": {
+                "active": {

```

Comment:
same as above about string enum instead of bool

Reply:
done